01/assets/js/app.js

```diff
@@ -1,4 +1,42 @@
+
 const a = '4.2';
 const b = 9;
+console.log(a, b);
+//typeof shows to me its str/number
+console.log(typeof a);
+console.log(typeof b);
+
+const c = parseInt(a);
+//działania arytmetyczne: + - * / %
+const res1 = c + b;
+const res2 = c - b;
+const res3 = c * b;
+const res4 = c / b;
+const res5 = c % b;
+
+console.log(res1, res2, res3, res4, res5);
+
+//funkcja if / else
+
+function funkcjaNr20(program) {
```

Comment:
Raczej nazwa parametru powinna wskazywać czym on jest `program` to zdecydowanie za ogólnie ;)